test/ForkRunnerTest.php

```diff
@@ -3,12 +3,16 @@
 
 class ForkRunnerTest extends \PHPUnit_Framework_TestCase
 {
-    public function testExampleFromTheDocs()
+    /**
+     * @param Collector $collector
+     * @dataProvider collectorsProvider
+     */
+    public function testExampleFromTheDocs(Collector $collector)
```

Comment:
I'd like to keep this test as it is, it is to make sure the code example from README.md is correct.